yb_migrate/cmd/common.go

```diff
@@ -135,6 +135,11 @@ func getMappingForTableNameVsTableFileName(dataDirPath string) map[string]string
 }
 
 func UpdateTableRowCount(source *srcdb.Source, exportDir string, tablesProgressMetadata map[string]*utils.TableProgressMetadata) {
+	var maxTableLines, totalTableLines int64
+
+	utils.InitJSON(exportDir)
```

Comment:
consider moving this call to a common location

Reply:
Moved to root